code/unclassified/binary_there_and_back_again

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#define x
+using namespace std;
+
+int binary_to_decimal(int n){
+    int sum=0,mul=1;
+while(n>0){
+    sum+=mul;
+    mul=mul*2;
+}
+}
+
+int decimal_to_binary(int n, int N){
+    sum=0;
```

Comment:
variable `sum` was not declared and several such errors exists. Kindly correct the code to get this merged 👍

Reply:
thanks 👍